README.rst

```diff
@@ -278,6 +278,16 @@ differ)::
 
     PASSED (skips=15, successes=1322)
 
+
+For the homeserver config, you can copy the sample config:
+
+    cp docs/sample_config.yaml homeserver.yaml
+
+Then finally start the app:
+
+    synctl start homeserver.yaml
```

Comment:
Generally I suggest using the demo scripts -- `./demo/start.sh` will give you 3 federated synapses running on 8080 - 8082.

Reply:
I assume this takes care of the config stuff too?

Reply:
Added as the initial suggestion :+1:

Reply:
Yeah it does everything for you.